app/views/accounting/search_transaction.html

```diff
@@ -48,7 +54,7 @@ <h4 class="control-label ng-binding">{{'label.heading.transactionid' | translate
                             placeholder="{{'label.input.searchbytransaction' | translate}}" class="form-control"/>
                         </div>&nbsp;&nbsp;
                         <div class="form-group">
-                            <a ng-click="searchTransaction()" class="btn btn-primary btn-space pull-left btn-toolbar" has-permission='READ_JOURNALENTRY'> Search <i class="fa fa-search "></i></a>
+                            <button type="submit" class="btn btn-primary btn-space pull-left btn-toolbar" has-permission='READ_JOURNALENTRY'> Search <i class="fa fa-search "></i></button>
```

Comment:
Since the button type is submit now, all the remaining button in the page should have type to button (like for parameters). Otherwise they will not work